src/frontend/screens/Game/GamePage/components/HLTBInline.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import GameContext from '../../GameContext'
+import { Speed, ExpandMore } from '@mui/icons-material'
+import { Accordion, AccordionSummary, AccordionDetails } from '@mui/material'
+import HowLongToBeat from 'frontend/components/UI/WikiGameInfo/components/HowLongToBeat'
+
+const HLTBInline = () => {
+  const { t } = useTranslation('gamepage')
+  const { wikiInfo } = useContext(GameContext)
+
+  const [isExpanded, setIsExpanded] = useState(false)
+
+  function handleChange() {
+    setIsExpanded((prevExpanded) => !prevExpanded)
+  }
+
+  if (!wikiInfo) {
+    return null
+  }
+
+  const howlongtobeat = wikiInfo.howlongtobeat
+
+  if (!howlongtobeat) {
+    return null
+  }
+
+  return (
+    <div className="hltbWrapper">
```

Comment:
I'm not sure which component is problematic, but when I move using a controller (or keyboard), the item for the HLTB info doesn't have the outline (the background is a biiiit darker)

https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/188464/4c207812-e47e-450a-a23b-3b5edbd0db5e

Reply:
Good catch, it's probably MUI stuff :/

Reply:
Fixed, it was kinda tricky, because of MUI